### DISCUSSION TITLE
[3.5RC1.1] More contrast or separation between multi-column field rows

### DISCUSSION THREAD

**Discussion by daltonrooney** [craftcms/cms#8869] [Ideas]
Love the new field designer, but I find it very hard to differentiate between certain kinds of fields with the new layout. Here's an example of a few different entry fields in 50/50 columns:

![Screen Shot 2020-07-22 at 10 56 53 AM](https://user-images.githubusercontent.com/742045/88212262-d5afdb80-cc0b-11ea-82aa-be56bfa689c0.png)

Immediate feedback from those in our office was that it's a bit visually disorienting. Adding even a tiny bit of visual separation between the fields goes a long way:

![Screen Shot 2020-07-22 at 11 05 40 AM](https://user-images.githubusercontent.com/742045/88212168-a8fbc400-cc0b-11ea-8305-ae742634313c.png)

**Comment by FreekVR** [comment]
Agree! Though I wouldn't add this for normal inputs, lightswitches etc which have clear visual distinction, but this would be great for relationship fields like assets/categories/entries.
